include/kwk/detail/sequence/prefilled.hpp

```diff
@@ -18,10 +18,26 @@ namespace kwk::__
 {
   struct empty_tuple
   {
-    static constexpr bool is_homogeneous = false;
+    static constexpr bool is_homogeneous = true;
     constexpr auto operator==(empty_tuple const&) const noexcept { return true; }
   };
 
+  // TODO: move to kumi
+  template <int index, typename... T>
+  using type_pack_element =
+#ifdef __clang__
+    __type_pack_element<index, T...>;
+#else
+    kumi::element_t<index, kumi::tuple<T...>>;
+#endif
+  template <int index, auto... NTTP>
+  using type_nttp_pack_element = type_pack_element<index, decltype(NTTP)...>;
```

Comment:
this needs a bunch of explanations cause atm I don't get what you're trying to fix with this.

Reply:
no fixing, was just extracting a repeated construct and using a faster/compiler builtin impl for it

Reply:
Per visioconf at 09/09 : This is somethign we want as a factoring tool. 
TODO: betetr name + better location